core-rust/mesh-api-server/src/mesh_api/conversions/common.rs

```diff
@@ -0,0 +1,123 @@
+use crate::prelude::*;
+
+pub fn to_mesh_api_operation_no_fee(
+    mapping_context: &MappingContext,
+    database: &StateManagerDatabase<impl ReadableRocks>,
+    index: i64,
+    status: &MeshApiOperationStatus,
+    account_address: &GlobalAddress,
+    resource_address: &ResourceAddress,
+    amount: Decimal,
+) -> Result<models::Operation, MappingError> {
+    // TODO:MESH what about fee locking, burning, minting?
+    let op_type = if amount.is_positive() {
+        MeshApiOperationTypes::Deposit
+    } else {
+        MeshApiOperationTypes::Withdraw
+    };
+    let currency =
+        to_mesh_api_currency_from_resource_address(mapping_context, database, resource_address)?;
+    let account = to_mesh_api_account_from_address(mapping_context, account_address)?;
+
+    // see https://docs.cdp.coinbase.com/mesh/docs/models#operation
+    Ok(models::Operation {
+        operation_identifier: Box::new(models::OperationIdentifier::new(index)),
+        related_operations: None,
+        _type: op_type.to_string(),
+        status: Some(status.to_string()),
+        account: Some(Box::new(account)),
+        amount: Some(Box::new(to_mesh_api_amount(amount, currency)?)),
+        coin_change: None,
+        metadata: None,
+    })
+}
+
+pub fn to_mesh_api_transaction_identifier(
```

Comment:
SUPER MINOR: Perhaps this should live in `block.rs` or `transaction.rs`? Or at least alongside the other similar conversions in `hashes.rs`

Reply:
Will do